File: web/app/Trip.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Trip extends Model
{
  protected $fillable = [
    'reference',
    'from_latitude',
    'from_longitude',
    'to_latitude',
    'to_longitude',
    'status',
    'user_id'
  ];

  public function user() {
    return $this->belongsTo(\App\User::class);
  }

  public function events() {
    return $this->belongsTo(\App\Event::class);
  }
}

File: android/app/src/main/java/tut/lbs/locationbasedsystem/models/Trip.java
package tut.lbs.locationbasedsystem.models;

import java.io.Serializable;

public class Trip implements Serializable{
    public long id;
    public String reference;
    public double from_latitude;
    public double from_longitude;
    public double to_latitude;
    public double to_longitude;
    public String status;
    public long user_id;
    public String created_at;
    public String updated_at;
    public User user;

}
